lib/cartopy/examples/gridliner.py

```diff
@@ -1,21 +1,17 @@
 """
 Gridlines and tick labels
 -------------------------
-
 These examples demonstrate how to quickly add longitude
 and latitude gridlines and tick labels on a non-rectangular projection.
-
 As you can see on the first example,
 longitude labels may be drawn on left and right sides,
 and latitude labels may be drawn on bottom and top sides.
 Thanks to the ``dms`` keyword, minutes are used when appropriate
 to display fractions of degree.
-
-
```

Comment:
I would not remove the blank lines between paragraphs, though maybe remove the second one here.